src/test/scala/akka/contrib/process/BlockingProcessSpec.scala

```diff
@@ -80,24 +100,16 @@ class Receiver(probe: ActorRef, stdinInput: immutable.Seq[String]) extends Actor
 
   override def receive: Receive = {
     case BlockingProcess.Started(stdin, stdout, stderr) =>
-      Source(stdinInput)
-        .map(ByteString.apply)
-        .runWith(Sink(stdin))
-      val stdoutFlow = Source(stdout).map(element => Out(element.utf8String))
-      val stderrFlow = Source(stderr).map(element => Err(element.utf8String))
-      val tellProbeSink = Sink.foreach(probe.!)
-      val graph =
-        FlowGraph.closed(tellProbeSink) { implicit b =>
-          resultSink =>
-            val merge = b.add(Merge[AnyRef](inputPorts = 2))
-            stdoutFlow ~> merge.in(0)
-            stderrFlow ~> merge.in(1)
-            merge ~> resultSink
-        }
-      graph
+      FlowGraph.closed(Sink.foreach(probe.!)) { implicit b =>
+        resultSink =>
+          val merge = b.add(Merge[AnyRef](inputPorts = 2))
+          Source(stdout).map(element => Out(element.utf8String)) ~> merge.in(0)
+          Source(stderr).map(element => Err(element.utf8String)) ~> merge.in(1)
+          merge ~> resultSink
+      }
         .run()
         .onComplete(_ => self ! "flow-complete")
-
+      Source(stdinInput).map(ByteString.apply).runWith(Sink(stdin))
```

Comment:
@huntc @ktoso This is interesting. Without running this line -after- running the graph above, there is some weird race-condition that will fail the build every 5-6 runs or so. Could this be indication of an issue with the bridges to Input and OutputStream?